### ISSUE TITLE
https doesn't work

### ISSUE THREAD

**Issue by tbhbestdevman** [michael-lynch/open-weather#16]
I am using this weather system for my website, it works on http but when I use https (default) it doesn't work.

**Comment by JamoCA** [comment]
I don't believe that it's mentioned, but I recall SSL requires a paid subscription.

Here's a solution someone else mentioned (using a SSL proxy):
https://forum.freecodecamp.org/t/local-weather-api-for-people-stuck-because-of-the-https-request/50274/4
